v2/features/dill/UnlockDate.tsx

```diff
@@ -1,32 +1,64 @@
-import React, { FC, useEffect, useState } from "react";
+import React, { FC, useState } from "react";
 import { Trans, useTranslation } from "next-i18next";
 import { IUserDillStats } from "picklefinance-core/lib/client/UserModel";
 import dayjs from "dayjs";
-
+import localizedFormat from "dayjs/plugin/localizedFormat";
+import duration from "dayjs/plugin/duration";
 import IncreaseLockDateModal from "v2/features/dill/IncreaseLockDateModal";
 import Button from "v2/components/Button";
-import DillCard from "./DillCard";
-
+dayjs.extend(duration);
```

Comment:
These plugins are already loaded into `dayjs` in `utils/day.js`, you can import it using `import dayjs from "util/dayjs";`